lib/dart/3.1.0/all.yaml

```diff
@@ -10,6 +10,7 @@ linter:
     - avoid_slow_async_io
     - avoid_type_to_string
     - avoid_types_as_parameter_names
+    - avoid_unstable_final_fields
```

Comment:
![must-badge](https://img.shields.io/badge/review-must-red.svg)
This seems wrong...
https://github.com/dart-lang/sdk/issues/53852